src/gradient_free_optimizers/optimizers/exp_opt/ensemble_optimizer.py

```diff
@@ -41,12 +33,7 @@ def __init__(
         epsilon=0.03,
         distribution="normal",
         n_neighbours=3,
-        estimators=[
```

Comment:
mutable defaults should never be used - general python thing, not specific to `sklearn` isolation